src/sidebar/util/test/sentry-test.js

```diff
@@ -25,6 +27,15 @@ describe('sidebar/util/sentry', () => {
   });
 
   describe('init', () => {
+    beforeEach(() => {
+      fakeDocumentReferrer = sinon.stub(document, 'referrer');
```

Comment:
I thought so too when I first wrote this and then found that it didn't work.

The `fakeDocumentReferrer` stub is needed to undo the stub at the end of the test with `fakeDocumentReferrer.restore()`. The usual `object.method.restore()` call doesn't work in this context because we've stubbed a property getter rather than an ordinary method.